kong/rest.lua

```diff
@@ -0,0 +1,41 @@
+local lapis      = require "kong-lapis"
+local get_method = ngx.req.get_method
+local header     = ngx.header
+local exit       = ngx.exit
+
+
+local ALLOWED_METHODS = {
+  "GET",
+  "HEAD",
+  "PUT",
+  "PATCH",
+  "POST",
+  "DELETE",
+}
+local METHODS = table.concat(ALLOWED_METHODS, ", ")
+
+
+do
```

Comment:
Yes you are right. I added another PR #2696 where I moved all this template refactoring.